developers_affiliations1.txt

```diff
@@ -15733,9 +15733,6 @@ LisaLudique: LisaLudique!users.noreply.github.com
 Listwon: Listwon!users.noreply.github.com
 	Independent until 2016-05-01
 	Pineapple Works from 2016-05-01
-LithiumSR: LithiumSR!users.noreply.github.com
-	Independent until 2021-04-01
-	Ericsson from 2021-04-01
 LittleBoy18: 2283985296!qq.com, littleboy18!users.noreply.github.com, zhaoze01!inspur.com
```

Comment:
But DevStats/gitdm tracks history, so it's OK to add a new GitHub login but we still need to have the old one, I'll add it back in post merge.

Reply:
Yes, because it exists in history - when we parse files from "GitHub archives" that old `id` will be there for files from, say, a months ago.